geoportailv3/static/js/locationinfo/locationinfodirective.js

```diff
@@ -298,6 +313,17 @@ app.LocationinfoController.prototype.showInfoPane_ =
         this['qrUrl'] = this.qrServiceUrl_ + '?url=' + shorturl;
       }, this));
 
+  // we need to wait for angular to be ready to manage animations
+  var dereg = this.scope_.$watch(goog.bind(function() {
+    return this.$animate_.enabled();
+  }, this), goog.bind(function(newVal) {
+    if (newVal) {
+      this.$timeout_(goog.bind(function() {
```

Comment:
I think the use of `$timeout` here deserves a comment in the code, even if why it's needed is not fully understood. We should be honest to ourselves :)